packages/node_modules/@ciscospark/widget-space/src/actions.js

```diff
@@ -154,7 +154,7 @@ export function getSpaceDetails({
 
     return sparkInstance.request({
       service: 'hydra',
-      resource: `/rooms/${spaceIdUUID}`
+      resource: `/rooms/${destinationId}`
```

Comment:
I thought you said this doesn't work with a request to `/rooms/<hydra encoded url>`?

Reply:
It doesn't work if it is a non 'US' cluster.